### PR TITLE
WIP: Video serializer

### PR DESCRIPTION
**Pull request by camsom**

### CODE REVIEW

**Comment by camsom** [comment]
@mparent61 @MichaelButkovic

File: zencoder/utils/serializers.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    class Meta:
+        placeholder_fields = {
+            field_name: placeholder_value
+        }
+    """
+
+    def to_representation(self, obj):
+        data = super(PlaceholderFieldsMixin, self).to_representation(obj)
+        # Check for conflicts & prioritize pre-existing values to placeholders.
+        meta = getattr(self, "Meta")
```
Comment (mparent61) [inline zencoder/utils/serializers.py]: What if class doesn't define `Meta`?

Comment (mparent61) [reply zencoder/utils/serializers.py]: Not a big deal, if this is only intended to be used by `ZencoderVideoSerializer`

Comment (camsom) [reply zencoder/utils/serializers.py]: I'll return a None, since the subsequent attribute also has a default.

**Review by mparent61 — APPROVED** [review]

**Pull request merged**